src/eos-autoupdater.c

```diff
@@ -66,6 +66,7 @@ static const char *UPDATE_STAMP_NAME = "eos-updater-stamp";
 static const char *CONFIG_FILE_PATH = SYSCONFDIR "/" PACKAGE "/eos-autoupdater.conf";
 static const char *OLD_CONFIG_FILE_PATH = SYSCONFDIR "/eos-updater.conf";
 static const char *STATIC_CONFIG_FILE_PATH = PKGDATADIR "/eos-autoupdater.conf";
+static const char *VAR_CONFIG_FILE_PATH = LOCALSTATEDIR "/local/" PACKAGE "/eos-autoupdater.conf";
```

Comment:
/usr/local is purely a symlink to /var/local when you've booted from ostree - so this will never really make a difference in Endless - but the "correct" approach I was imagining was an XDG_DATA_DIR style search order like /etc/foo, /usr/local/etc/foo, /usr/etc/foo. Referring to /var for configuration just upsets my Debian sensibilities. It's ostree's business to move stuff around there.

You are totally right that, as /var/local is in fact just /usr/local relocated, it should have the same sub-heirachy as /usr. I suggest /usr/local/etc because we have the ostree convention of our "default" /etc being in /usr/etc, so searching /usr/local/etc isn't a bad shout.